keyboards/ml/gas75/keymaps/default/keymap.c

```diff
@@ -159,23 +159,43 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
 }
 
 #ifdef ENCODER_ENABLE
-bool encoder_update_user(uint8_t index, bool clockwise) {
-    if (index == 0) {
-        if (layer_state_is(_BASE)) {
-            if (clockwise) {
-                tap_code(KC_VOLU);
-            } else {
-                tap_code(KC_VOLD);
-            }
-        } else if (layer_state_is(_FN)) {
-            if (clockwise) {
-                rgb_matrix_step();
-            } else {
-                rgb_matrix_step_reverse();
-            }
+
+#define ENCODERS 1
```

Comment:
Honestly, I'd really prefer to keep this out of the default keymap. 

Especially since, end of month, encoder map lands, and provides a better and official way to handle this.

Reply:
Thanks for pointing it, I've push a new commit to revert back to old one